pages/index.js

```diff
@@ -14,9 +14,7 @@ const StyledLink = styled(Link)`
 export default function HomePage() {
   return (
     <Flex justifyContent="center">
-      <StyledLink href="/marketplace">
-        Welcome to the Star Wars Bazaar!
-      </StyledLink>
+      <StyledLink href="/login">Welcome to the Star Wars Bazaar!</StyledLink>
```

Comment:
Please rethink this concept. When a user opens your app in the browser, the following happens:

- There is no clear indication that the user needs to login as the login is hidden behind the greeting "Welcome to the Star Wars Bazaar!"
- If the user does not get, how to log in, they can still access all pages of your app. If the user opens the profile page, the page looks somewhat broken to the user as there is no user data displayed (no picture etc.)

👉🏼 Therefore I suggest the following:

- There should be clear instructions to log in, e.g. a login-button beneath the greeting.
- The user should not have access to certain parts (user profile) of your application without logging in. Or you can check on the profile page if the user is logged in. If they aren't, you can render a login-button instead of the content of the profile page.
- If there is a login, there should also be a logout function.